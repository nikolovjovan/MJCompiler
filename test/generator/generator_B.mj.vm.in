1
8
-5 8 2 97 25 15 9 -7
15
