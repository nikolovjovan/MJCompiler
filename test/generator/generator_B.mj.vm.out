1:0
2:0
3:0
4:0
5:-6
6:1
7:
8:3
switch = b
n = arr[0] = arr[1] = arr[2] = arr[3] = arr[4] = arr[5] = arr[6] = arr[7] = arr[0] = -5
arr[1] = 8
arr[2] = 2
arr[3] = 97
arr[4] = 25
arr[5] = 15
arr[6] = 9
arr[7] = -7
-7 -5 2 8 9 15 25 97 
compare
for true
foreach true
e = for globalI = 5
true
foreach it = 15
true
odd count
for 6
foreach 6

Completion took 0 ms